### PR TITLE
Add makefile for reconciler

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Add makefile for reconciler

**Related issue(s)**

See https://github.com/kyma-incubator/reconciler/issues/234

### CODE REVIEW

File: tools/reconciler/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+NAME = reconciler
```
Comment (pPrecel) [inline tools/reconciler/Makefile:1]: What do you think about making the Reconciler Team CODEOWNERS for this new directory?

Comment (suleymanakbas91) [reply tools/reconciler/Makefile:1]: Done

**Review by pPrecel — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -43,3 +43,12 @@ Kyma Control Plane, as a part of Kyma, uses [Octopus](https://github.com/kyma-in
 ```
 
 Read [this](https://kyma-project.io/docs/root/kyma#details-testing-kyma) document to learn more about testing in Kyma.
+
+## Reconciler
+
+If you want to deploy only the [Reconciler](https://github.com/kyma-incubator/reconciler) for testing purposes:
```
Comment (mmitoraj) [inline README.md:49]:
```suggestion
If you want to deploy only the [Reconciler](https://github.com/kyma-incubator/reconciler) for testing purposes, run:
```

**Review by mmitoraj — APPROVED** [review]

**Review by pPrecel — APPROVED** [review]

**Review by franpog859 — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**